client/src/components/History/GanttVisualisation/Gantt.vue

```diff
@@ -1,13 +1,48 @@
 <template>
     <div>
         <svg id="gantt"></svg>
-        <button id="view" @click="changeQDayView">Quarter Day View</button>
-        <button id="view" @click="changeHDayView">Half Day View</button>
-        <button id="view" @click="changeDayView">Day View</button>
-        <button id="view" @click="changeWeekView">Week View</button>
-        <button id="view" @click="changeMonthView">Month View</button>
-        <button id="view" @click="changeHourView">Hour View</button>
-        <button id="view" @click="changeMinuteView">Minute View</button>
+        <div class="sticky">
+            <button id="QDayView" @click="changeQDayView">Quarter Day View</button>
+            <button id="HDayView" @click="changeHDayView">Half Day View</button>
+            <button id="dayView" @click="changeDayView">Day View</button>
+            <button id="weekView" @click="changeWeekView">Week View</button>
+            <button id="monthView" @click="changeMonthView">Month View</button>
+            <button id="hourView" @click="changeHourView">Hour View</button>
+            <button id="minuteView" @click="changeMinuteView">Minute View</button>
+            <button class="btn btn-info" @click="showModal">show modal</button>
+            <div v-if="myModal">
```

Comment:
Lets have the reactive variable named something like ModalOpen or openModal etc. it becomes more understandable